manifests/golang.yml

```diff
@@ -468,6 +468,18 @@ tests/:
     test_otel_sdk_interoperability.py: missing_feature
     test_otel_span_with_baggage.py:
       Test_Otel_Span_With_Baggage: missing_feature
+    test_parametric_endpoints.py:
+      Test_Parametric_DDSpan_Add_Link: missing_feature (add_link endpoint is not implemented)
+      Test_Parametric_DDSpan_Set_Resource: missing_feature (set_resource endpoint is not implemented)
+      Test_Parametric_DDTrace_Baggage: missing_feature (baggage is not supported)
+      Test_Parametric_DDTrace_Crash: missing_feature (crash endpoint is not implemented)
+      Test_Parametric_DDTrace_Current_Span: missing_feature (current span endpoint is not defined)
+      # The set_name endpoint should set the resouce name (not the span name)
+      Test_Parametric_OtelSpan_Set_Name: bug (APMAPI-778)
+      # String attributes are incorrectly stored/serialized in a list
+      Test_Parametric_OtelSpan_Start_Finish: bug (APMAPI-795)
```

Comment:
They are? If you have more info, can we link to an actual dev card?

Reply:
Ya we can after we confirm whether this is a bug with the golang tracer or the parametric app